### PR TITLE
Feat/veeam backup terminate

### PR DESCRIPTION
**Pull request by radireddy**
Terminate veeam backup

#MANAGER-4450

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "datacenter_backup_delete_title": "Désactivation de la sauvegarde",
+  "datacenter_backup_delete_description": "Souhaitez-vous désactiver la sauvegarde pour le datacenter {{datacenter}}? Vous devrez vérifier la désactivation de la sauvegarde pour chaque machine virtuelle depuis vSphere. Attention, cette action supprimera définitivement toutes vos sauvegardes.",
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/translations/Messages_fr_FR.json]:
```suggestion
  "datacenter_backup_delete_description": "Souhaitez-vous désactiver la sauvegarde pour le datacenter {{datacenter}} ? Vous devrez vérifier la désactivation de la sauvegarde pour chaque machine virtuelle depuis vSphere. Attention, cette action supprimera définitivement toutes vos sauvegardes.",
```
non-breaking space

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "datacenter_backup_delete_cancel": "Annuler",
+  "datacenter_backup_delete_confirm": "Désactiver la sauvegarde",
+  "datacenter_backup_delete_success": "Votre solution de sauvegarde a bien été désactivée.",
+  "datacenter_backup_delete_error": "Une erreur s'est produite lors de la désactivation de votre service de sauvegarde: {{message}}"
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/translations/Messages_fr_FR.json]:
```suggestion
  "datacenter_backup_delete_error": "Une erreur s'est produite lors de la désactivation de votre service de sauvegarde : {{message}}"
```
non-breaking space

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+import '@ovh-ux/manager-core';
+
+const moduleName = 'ovhManagerDedicatedCloudBackupDeleteLazyloading';
+
+angular
+  .module(moduleName, ['ui.router', 'ovhManagerCore', 'oc.lazyLoad'])
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/delete/index.js]: Order please

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#9715.0 ✘  
* 
  * Build ✘

**Pull request closed without merging**